### ISSUE TITLE
Add a flutterw command to install in another directory

### ISSUE THREAD

**Issue by dalewking** [passsy/flutter_wrapper#20]
Been using flutter wrapper in multiple projects and when starting a new project it is a bit of an inconvenience to have to go to the project page to find the shell command to install it in the new project. Since I already have it installed in another project it would be nice to be able to leverage that existing install to install in another directory using an easy to remember command.

My thought was going to the existing project directory and being able to say something like:

    ./flutterw install /path/to/new/project

**Comment by neiljaywarner** [comment]
@dalewking  have you ever used this with windows?
hope you are enjoying flutter

**Comment by dalewking** [comment]
No, and I don't do much flutter and quit using flutterw when I do

**Comment by neiljaywarner** [comment]
Well I apologize for asking here. Please consider removing the comment as a kindness to passy or perhaps rephrase remind as a bit report and or feature request but anyway I like it except for missing window s support for those unlucky folks. Nice running into you though

**Comment by dalewking** [comment]
What I ended up doing is adding this to my .bash_profile:

```
alias fwinstall='sh -c "$(curl -fsSL https://raw.githubusercontent.com/passsy/flutter_wrapper/master/install.sh)'
```
